lib/src/api.ts

```diff
@@ -32,6 +14,7 @@ export type PayloadRequestData = {
   fromAccount: Account;
   toAccount: Account;
   amount: BigNumber;
+  amountInAtomicUnit: bigint;
```

Comment:
Why do we need both of them?

Reply:
Good question :) 
I will need to check with backends but maybe we can remove it. Thanks :)